handbook/engineering/hiring/software-engineer-coding-exercise.md

```diff
@@ -25,9 +25,16 @@ We designed this exercise to measure your understanding of building web UIs.
 - You will choose a three hour timeframe to independently work on the exercise. You can choose any day and time; we just schedule an email.
 - The exercise asks you to implement a UI in a scaffold TypeScript React app.
 - We provide you with the code of the scaffold app, the URL of a working GraphQL API to fetch data from and a link to a Figma design for the UI.
-- You are free in how you chose to implement it and can change the existing code in any way you'd like.
+- You are free in how you chose to implement your solution, and can change the existing code in any way you'd like, however be aware the scaffold app is there to save you time.
```

Comment:
This seems to contradict your other change to require React, because it says the scaffold can be changed in any you like (which would include changing it to not use React). Is that intentional? In any case we should realign the two documents to not be contradictory.

Reply:
I've removed this statement as I don't think it adds any value to the candidate based on the current expectation we outline.

Reply:
Sounds good. Maybe the only thing we would wanna keep communicate is "you can add dependencies" because I think that's a common question for coding exercises